daffodil-core/src/main/scala/org/apache/daffodil/dsom/ChoiceGroup.scala

```diff
@@ -222,7 +227,7 @@ abstract class ChoiceTermBase(
               event.qname, trds.map { trd => "%s at %s".format(trd.diagnosticDebugName, trd.locationDescription) }.mkString("\n"))
           } else {
             val eventType = event match {
-              case _: ChoiceBranchEndEvent => "end"
+              case _: ChoiceBranchEndEvent   => "end"
```

Comment:
Using default indentation settings, we traded one set of whitespace changes for another, which is to align the arrows, and various other things.

Reply:
I've fixed these now.